tests/test_component.py

```diff
@@ -181,6 +181,7 @@ def setup(self, *, flag, value):
 
         def transform(self, dataframe):
             assert isinstance(dataframe, pd.DataFrame)
+            return dataframe.rename(columns={"images": "embeddings"})
```

Comment:
Not completely sure what you mean. I had to change this line since the `Component` otherwise fails, as the Dask `map_partitions` method fails if the returned dataframe doesn't match the meta dataframe we pass to it. Since the meta dataframe is built based on the component spec, it already checks the columns against the spec.

Reply:
ok that's clear thanks :)